<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>377</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>신규 지급계좌 등록</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>49</y>
     <width>641</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>56</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>사용자 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>56</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>108</y>
     <width>95</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>지급계좌명칭</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="account_name_widget">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>101</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_account_button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>149</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>신규등록 추가</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="registed_account_tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>158</y>
     <width>161</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>128</y>
     <width>641</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>349</y>
     <width>631</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="new_account_tableWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>159</y>
     <width>321</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>계좌명</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>은행명</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>계좌번호</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="account_save_button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>271</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>저장(F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="account_close_button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>309</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="account_view_button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>189</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>다시 보기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selected_row_delete_button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>229</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택 행 삭제</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>108</y>
     <width>56</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>은행명</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bank_name_widget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>108</y>
     <width>63</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>계좌번호</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_account_widget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>99</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>139</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>기존 등록된 사항</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>신규등록</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>74</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>구 분</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gubun_comboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>account_name_widget</tabstop>
  <tabstop>bank_name_widget</tabstop>
  <tabstop>new_account_widget</tabstop>
  <tabstop>add_account_button</tabstop>
  <tabstop>account_save_button</tabstop>
  <tabstop>account_view_button</tabstop>
  <tabstop>account_close_button</tabstop>
  <tabstop>registed_account_tableWidget</tabstop>
  <tabstop>new_account_tableWidget</tabstop>
  <tabstop>selected_row_delete_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>add_account_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>account_reg_input()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>account_view_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>registed_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selected_row_delete_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>delete_selected_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>account_save_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>file_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>account_close_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>file_save_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>618</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>account_reg_input()</slot>
  <slot>file_save()</slot>
  <slot>file_save_cancel()</slot>
  <slot>registed_data()</slot>
  <slot>delete_selected_row()</slot>
 </slots>
</ui>
